glusterd2/device/store-utils.go

```diff
@@ -0,0 +1,47 @@
+// Package device stores device information in the store
+package device
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/gluster/glusterd2/glusterd2/store"
+	"github.com/gluster/glusterd2/pkg/api"
+)
+
+const (
+	devicePrefix string = "devices/"
+)
+
+// GetDevice returns devices of specified peer from the store
+func GetDevice(peerid string) (*api.Device, error) {
```

Comment:
This must be `GetDevices` and must return a slice of devices and not a single device.